.github/workflows/docs-upstream.yml

```diff
@@ -30,6 +28,6 @@ on:
 
 jobs:
   validate:
-    uses: docker/docs/.github/workflows/validate-upstream.yml@main
+    uses: docker/docs/.github/workflows/validate-upstream.yml@919a9b9104a34a40b30d116529bcce589a544d1c  # pin for artifact v4 support: https://github.com/docker/docs/pull/19220
```

Comment:
Is this because other branches / dependencies  are not yet updated? (sorry perhaps silly question; trying to understand why `main` wouldn't work if both `docs` and this repository now support v4)

Reply:
This is to avoid having the workflow failing similar to https://github.com/docker/buildx/pull/2210#issuecomment-1911663381 in case artifact actions are bumped to v5 in the future on docs main and would/could therefore be incompatible with current v4.

Reply:
Ah, it's to pin to the _current_ version (prevent updates); I was reading and thought it was downgrading the version 🤗